packages/askui-nodejs/src/lib/interactive_cli/cli.ts

```diff
@@ -0,0 +1,149 @@
+import inquirer, { QuestionCollection } from 'inquirer';
+import { Command, Option, OptionValues } from 'commander';
+import { CreateExampleProject } from './create-example-project';
+import { CliOptions } from './cli-options-interface';
+
+const nonEmpty = (subject: string) => (input: string) => (input.trim().length > 0) || `${subject} is required.`;
+
+const questions: QuestionCollection = [
+  {
+    type: 'list',
+    name: 'operatingSystem',
+    message: 'Which operating system are you on?',
+    choices: ['windows', 'linux', 'macos'],
+  },
+  {
+    type: 'list',
+    name: 'progLanguage',
+    message: 'Which language do you prefer?',
+    choices: ['typescript'],
```

Comment:
Do we need this or did you miss some choices here. like `javascript`

Reply:
I missed that while I was copying it over from an old branch. Good catch!